support/camel-k-maven-logging/pom.xml

```diff
@@ -30,10 +30,9 @@
 
     <properties>
       <!-- Maven structural logging -->
-      <logback-version>1.2.3</logback-version>
-      <logstash-logback-version>4.11</logstash-logback-version>
-      <jackson-version>2.9.10</jackson-version>
-      <jackson-databind-version>2.9.10.8</jackson-databind-version>
+      <logback-version>1.2.11</logback-version>
+      <logstash-logback-version>7.2</logstash-logback-version>
+      <jackson-version>2.13.3</jackson-version>
```

Comment:
jackson-core and jackson-databind versions might differ. Is it really ok to combine them?

Reply:
I think in general these two should align. There have been quick fixes to jackson-databind in the past that made versions diverge but I think this should not be the usual case.

Reply:
@davsclaus @lburgazzoli WDYT?

Reply:
LGTM, we can add a dedicate property when/if it is really needed